backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc

```diff
@@ -13,3 +13,14 @@ The default plug-ins enable Velero to integrate with certain cloud providers and
 
 include::modules/oadp-features.adoc[leveloffset=+1]
 include::modules/oadp-plugins.adoc[leveloffset=+1]
+
+== OADP support for IBM Power Systems and IBM Z
+
+OADP is platform neutral. The information that follows relates only to IBM Power Systems and to IBM Z systems.
```

Comment:
@RichardHoch 
Maybe drop 'systems' at the end. Otherwise, it looks strange that the first 'Systems' is capitalized and the second is not.